packages/admin-next/dashboard/src/i18n/translations/en.json

```diff
@@ -924,8 +924,12 @@
       "inboundShippingHint": "Choose which method you want to use.",
       "returnableQuantityLabel": "Returnable quantity",
       "refundableAmountLabel": "Refundable amount",
-      "returnRequestedInfo": "{{requestedItemsCount}}x item return requested",
-      "returnReceivedInfo": "{{requestedItemsCount}}x item return received",
+      "returnRequestedInfo": "{{requestedItemsCount}}x items return requested",
+      "returnReceivedInfo": "{{requestedItemsCount}}x items return received",
+      "itemReceived": "Items received",
+      "returnRequested": "Return requested",
+      "damagedItemReceived": "Damaged items received",
+      "damagedItemsReturned": "{{quantity}}x Damaged items returned",
```

Comment:
suggestion/nit: can "Damaged" be lower first letter, it looks a bit off next to the return received line